ees_sharepoint/sync_sharepoint.py

```diff
@@ -117,8 +117,10 @@ def fetch_sites(self, parent_site_url, sites, ids, index, start_time, end_time):
         :param index: index, boolean value
         :param start_time: start time for fetching the data
         :param end_time: end time for fetching the data
+        :param document_list: list of document of sites
         Returns:
-            document: response of sharepoint GET call, with fields specified in the schema
+            sites: list of site paths
+            documents: response of sharepoint GET call with fields specified in the schema
         """
```

Comment:
I don't quite understand why there's a need to both return documents and also pass `document_list`. Can the function just collect documents returned from `fetch_sites` calls for each sub-site and return it along with the documents fetched for this site?

Reply:
Sure, i'll update it.